From hjk@linutronix.de  Tue Feb 19 14:02:23 2008
From: Hans-Jürgen Koch <hjk@linutronix.de>
Date: Tue, 19 Feb 2008 21:07:19 +0100
Subject: UIO: arch/arm/Kconfig: Make UIO available on ARM architecture
To: greg@kroah.com
Message-ID: <20080219210719.7c14d3eb@dilbert.local>

Source drivers/uio/Kconfig to make UIO available in menuconfig if ARCH=arm.

Signed-off-by: Hans J Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/arm/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1172,6 +1172,8 @@ source "drivers/dma/Kconfig"
 
 source "drivers/dca/Kconfig"
 
+source "drivers/uio/Kconfig"
+
 endmenu
 
 source "fs/Kconfig"
